doc/ssl/SSL_SESSION_get_time.pod

```diff
@@ -36,6 +40,9 @@ to B<tm>.
 The SSL_get_time(), SSL_set_time(), SSL_get_timeout(), and SSL_set_timeout()
 functions are synonyms for the SSL_SESSION_*() counterparts.
 
+SSL_SESSION_get0_hostname returns the pointer to the hostname for session B<s>.
```

Comment:
Right, so the text needs to be a bit more explicit about what hostname we are talking about.